README.md

```diff
@@ -70,6 +70,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing. F
  - [Builder Book](https://builderbook.org) - Learn how to build a full stack JavaScript web app from scratch
  - [MongoDB Applied Design Patterns (Rick Copeland)](https://www.oreilly.com/library/view/mongodb-applied-design/9781449340056/)
  - [The Little MongoDB Book](https://www.openmymind.net/2011/3/28/The-Little-MongoDB-Book/) - Basic introduction
+ - [Practical MongoDB Aggregations](https://www.practical-mongodb-aggregations.com/) - Howw to develop effective and optimal data manipulation and analytics pipelines"
```

Comment:
I hope that you pay more attention to typos in your book. :P 

```suggestion
 - [Practical MongoDB Aggregations](https://www.practical-mongodb-aggregations.com/) - How to develop effective and optimal data manipulation and analytics pipelines
```

Also, entries in the list should be ordered alphabetically.